tutorials/tour/abstract-types.md

```diff
@@ -5,11 +5,10 @@ title: Abstract Types
 disqus: true
 
 tutorial: scala-tour
-num: 2
-outof: 35
```

Comment:
There are only 33 pieces to the tutorial, not 35. I don't even really know what effect `outof` has but I figured it should go in the first section.